pkg/util/filter/schema.go

```diff
@@ -17,39 +17,20 @@ package filter
 import (
 	"strings"
 
-	"github.com/pingcap/tidb/pkg/parser/mysql"
+	"github.com/pingcap/tidb/pkg/meta/metadef"
 )
 
 var (
 	// DMHeartbeatSchema is the heartbeat schema name
-	DMHeartbeatSchema = "DM_HEARTBEAT"
-	// InformationSchemaName is the `INFORMATION_SCHEMA` database name.
-	InformationSchemaName = "INFORMATION_SCHEMA"
-	// PerformanceSchemaName is the `PERFORMANCE_SCHEMA` database name.
-	PerformanceSchemaName = "PERFORMANCE_SCHEMA"
-	// MetricSchemaName is the `METRICS_SCHEMA` database name.
-	MetricSchemaName = "METRICS_SCHEMA"
+	DMHeartbeatSchema = "dm_heartbeat"
 	// InspectionSchemaName is the `INSPECTION_SCHEMA` database name
-	InspectionSchemaName = "INSPECTION_SCHEMA"
-	// WorkloadSchemaName is the `WORKLOAD_SCHEMA` database name
-	WorkloadSchemaName = strings.ToUpper(mysql.WorkloadSchema)
+	InspectionSchemaName = "inspection_schema"
```

Comment:
those 2 are used by DM test only, change to lower case should be fine

https://github.com/search?q=repo%3Apingcap%2Ftiflow+DMHeartbeatSchema&type=code

https://github.com/search?q=repo%3Apingcap%2Ftiflow+InspectionSchemaName&type=code